engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/CanBeSubclassDeterminer.scala

```diff
@@ -49,7 +49,17 @@ private[typed] object CanBeSubclassDeterminer {
     //throw validation errors in this case. It's better to accept to much than too little
       superclassCandidate.params.zip(givenClass.params).forall(t => canBeSubclassOf(t._1, t._2) || canBeSubclassOf(t._2, t._1))
 
-    givenClass == superclassCandidate || ClassUtils.isAssignable(givenClass.klass, superclassCandidate.klass) && hasSameTypeParams
+    val canBeSubclass = givenClass == superclassCandidate || ClassUtils.isAssignable(givenClass.klass, superclassCandidate.klass) && hasSameTypeParams
+    canBeSubclass || canBeConvertedTo(givenClass, superclassCandidate)
+  }
+
+  private def canBeConvertedTo(givenClass: TypedClass, superclassCandidate: TypedClass): Boolean = {
```

Comment:
maybe some better name? Also wonder what happens when we use non-spel expressions that don't have such type conversions?? let's add at least some comment about it?

Reply:
Can't find better name. This method check if some (safe) conversion is possible. In evaluation it is also named "conversion". What is your proposal for name? I've added some comment, can you check if it is all right?